src/raven.js

```diff
@@ -1423,8 +1423,9 @@ Raven.prototype = {
     var testString = (type || '') + ': ' + (message || '');
 
     if (
-      !!this._globalOptions.ignoreErrors.test &&
-      this._globalOptions.ignoreErrors.test(testString)
+      !!this._globalOptions.ignoreErrors.test && (
+        this._globalOptions.ignoreErrors.test(message) ||
+        this._globalOptions.ignoreErrors.test(testString))
```

Comment:
Can we also change it so that `testString` doesn't become `:message` if it's undefined? per https://github.com/getsentry/raven-js/issues/1075#issuecomment-334356750

Reply:
Yep! Addressed.

Reply:
This code will double-check the same string if `type` is not defined, but I think we can live with it. 
An alternative would be something like this:

```js
var shouldIgnore = false

if (!!this._globalOptions.ignoreErrors.test) {
  shouldIgnore = this._globalOptions.ignoreErrors.test(message)
  if (!shouldIgnore) shouldIgnore = this._globalOptions.ignoreErrors.test(testString)
}
```

which doesn't really improve a readability.